### PR TITLE
chore: exclude pandas 2.2.0 to unblock prerelease tests

### PR DESCRIPTION
**Pull request by shobsi**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes internal issue 322048974 🦕

### CODE REVIEW

**Review by ashleyxuu — APPROVED** [review]

File: noxfile.py [hunk 1]
```diff
@@ -542,7 +542,12 @@ def prerelease(session: nox.sessions.Session, tests_path):
         # TODO(shobs): Remove excluding version 2.2.0rc0 after
         # https://github.com/pandas-dev/pandas/issues/56646 and
         # https://github.com/pandas-dev/pandas/issues/56651 are resolved.
-        "pandas!=2.1.4,!=2.2.0rc0",
+        #
+        # TODO(shobs): Remove excluding version 2.2.0 after
+        # https://github.com/googleapis/python-bigquery-dataframes/issues/341
+        # https://github.com/googleapis/python-bigquery-dataframes/issues/337
+        # are resolved
+        "pandas!=2.1.4, !=2.2.0rc0, !=2.2.0",
```
Comment (chelsea-lin) [inline noxfile.py:550]: nit: to avoid other pandas newer version failures, can we set it as `pandas<=2.1.3` or `pandas=2.1.3`?

**Review by chelsea-lin — APPROVED** [review]
Overall looks good to me.

File: noxfile.py (hunk 1 above)
Comment (shobsi) [reply noxfile.py:550]: Yeah it is intentional. See for reference https://github.com/googleapis/python-bigquery-dataframes/pull/268/files/911109041e87070b96738048bbcc6874ba25ac31#r1423205172

Comment (chelsea-lin) [reply noxfile.py:550]: cool! good to know!

**Pull request merged**